{"name": "test_search_with_invalid_date_range", "status": "failed", "statusDetails": {"message": "AssertionError: assert 200 == 400\n +  where 200 = <Response [200]>.status_code", "trace": "self = <test_articles.TestArticlesAPI object at 0x00000148D0B79790>, session = <requests.sessions.Session object at 0x00000148CFB5A310>, api_key = '3iElcEAbAjbQuSzc3jRJtU8JVTnA7x5g'\napi_base_url = 'https://api.nytimes.com/svc/'\n\n    def test_search_with_invalid_date_range(self, session, api_key, api_base_url):\n        params = {\n            \"api-key\": api_key,\n            \"q\": \"Python\",  # Замените на свой запрос\n            \"sort\": \"newest\",\n            \"begin_date\": \"2023-01-01\",  # Некорректный формат даты, должен вернуть ошибку\n            \"end_date\": \"2023-12-31\"\n        }\n        response = session.get(f\"{api_base_url}{self.endpoint}\", params=params)\n    \n>       assert response.status_code == 400\nE       assert 200 == 400\nE        +  where 200 = <Response [200]>.status_code\n\ntests\\test_articles.py:70: AssertionError"}, "start": 1690385824795, "stop": 1690385825241, "uuid": "4950349b-b014-4142-8d8f-0572aada8096", "historyId": "fc2413239be63261f965d4ecdec61ebe", "testCaseId": "fc2413239be63261f965d4ecdec61ebe", "fullName": "tests.test_articles.TestArticlesAPI#test_search_with_invalid_date_range", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_articles"}, {"name": "subSuite", "value": "TestArticlesAPI"}, {"name": "host", "value": "Chris"}, {"name": "thread", "value": "26012-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_articles"}]}